src/gui/menu/description.py

```diff
@@ -30,13 +30,23 @@ def setup(self):
         self.description_surface = pygame.Surface(
             self.description_rect.size, pygame.SRCALPHA
         )
+        self.description_surface.fill('green')
+        self.description_surface.set_colorkey('green')
```

Comment:
Not necessary

Reply:
I get it but why pygame.SRCALPHA doesn't work so ?

Reply:
The returned surface is already transparent when created